### DISCUSSION TITLE
NSAIDs in the ICU

### DISCUSSION THREAD

**Discussion by aegis301** [MIT-LCP/mimic-code#1445] [MIMIC-General]
Dear MIMIC community,

I struggle to find any NSAID in the mimic_icu.inputevents table. Is this likely that no patients in the ICU received any NSAIDs? Does anyone here have experience in this topic? Or am I using the wrong table?

Kind regards

**Comment by marymlucas** [comment]
Hi @aegis301 are you looking for a particular NSAID? I found this interesting read on use of NSAIDs on ICU patients, which might explain your findings... https://emcrit.org/ibcc/pain/#NSAIDs
